reference/rebac_admin_backend.rst

```diff
@@ -0,0 +1,32 @@
+ReBAC Admin Backend
+=======================
+
+The following sections briefly describe the functionalities offered by the ReBAC Admin Backend.
+
+OpenAPI specification
+---------------------
+The OpenAPI spec can be found at this endpoint for your deployment: `https://<jimm-deployment>/v1/rebac/swagger.json`
+
+Authentication
+--------------
+These endpoints are meant to be called from a web browser, therefore the authentication is handled via Cookies.
+
+JAAS Implementation
+-------------------
+
+We implement a subset of the operations described in the OpenAPI spec. 
+
+ ====== ============= =========================================================================
+ Status Entities      Notes
+ ====== ============= =========================================================================
+   ✅   capabilities     
+   ✅   entitlements     
+   ✅   capabilities     
+   ✅   groups     
+   ✅   resources     
+   🟡   identities    no support for creation, update and deletion.
+   ❌   roles         no support for roles.
+ ====== ============= =========================================================================
```

Comment:
The entries in the second column should be enclosed in double backticks and with a leading slash (`/`) to resemble endpoint realms (e.g., \`\`/capabilitiles\`\`).